CHANGELOG.md

```diff
@@ -4,6 +4,43 @@
 `psql` with the `-X` flag to prevent any `.psqlrc` commands from
 accidentally triggering the load of a previous DB version.**
 
+## 1.2.0 (2019-01-29)
+
+This release adds bugfixes, and optimizations such as ordered append.
+
+**This is our first Enterprise release!**
+
+This release adds code under a new license, LICENSE-TIMESCALE. This code can be
+found in `tsl`, and contains scheduled tasks, such as a scheduled `drop_chunks`
+and `reorder`.
+
+**Notable commits**
+
+[fa61613]Change time_bucket_gapfill argument names
+[fdaa717] Update telemetry with prettier os info
```

Comment:
I think we can remove this

Reply:
Also need to convert this section into a markdown list so it renders correctly

Reply:
I always forget that.